package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@modelcontextprotocol/sdk",
-  "version": "1.5.0",
+  "version": "1.7.0",
   "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "@modelcontextprotocol/sdk",
-      "version": "1.5.0",
```

Comment:
this is due to the `package.lock` not being updated on the last release. not sure if you want this bumped for the fix or not